core/storage/classifier/gae_models.py

```diff
@@ -114,6 +116,10 @@ class ClassifierTrainingJobModel(base_models.BaseModel):
     # The list contains dicts where each dict represents a single training
     # data group.
     training_data = ndb.JsonProperty(default=None)
+    # The time when the job was picked up last time.
```

Comment:
The time when the job's status should next be checked. (or something like that?)